docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,7 @@ Version history
 * access log: added FILTER_STATE :ref:`access log formatters <config_access_log_format>` and gRPC access logger.
 * api: remove all support for v1
 * build: official released binary is now built against libc++.
+* logger: added `--log-format-escaped` command line option to escape newline characters in application logs.
```

Comment:
As long as you are in here do you mind ref linking to the option docs? See other examples in this file. Thank you!

/wait

Reply:
Done